tests/pactHelper.js

```diff
@@ -1,11 +1,19 @@
 var config = require('./config/config.json')
 const utf8 = require('utf8')
+const { util } = require('util')
 
 var validUserPasswordHash = btoa(config.username + ':' + config.password)
 const Pact = require('@pact-foundation/pact-web')
 
 const accessControlAllowHeaders = 'OC-Checksum,OC-Total-Length,OCS-APIREQUEST,X-OC-Mtime,Accept,Authorization,Brief,Content-Length,Content-Range,Content-Type,Date,Depth,Destination,Host,If,If-Match,If-Modified-Since,If-None-Match,If-Range,If-Unmodified-Since,Location,Lock-Token,Overwrite,Prefer,Range,Schedule-Reply,Timeout,User-Agent,X-Expected-Entity-Length,Accept-Language,Access-Control-Request-Method,Access-Control-Allow-Origin,ETag,OC-Autorename,OC-CalDav-Import,OC-Chunked,OC-Etag,OC-FileId,OC-LazyOps,OC-Total-File-Length,Origin,X-Request-ID,X-Requested-With'
+const defaultAccessControlAllowMethods = 'GET,OPTIONS,POST,PUT,DELETE,MKCOL,PROPFIND,PATCH,PROPPATCH,REPORT'
 const origin = 'http://localhost:9876'
+const ocsCapabilitiesPathMatcher = '.*\\/ocs\\/v(1|2)\\.php\\/cloud\\/capabilities'
+const applicationOrJson = 'application/json; charset=utf-8'
```

Comment:
the two sides of the mime type do not mean `or` https://en.wikipedia.org/wiki/Media_type#Naming its a type and a subtype